src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -665,7 +664,7 @@ class DocBaseViewer extends BaseViewer {
         this.pdfLinkService = new this.pdfjsViewer.PDFLinkService({
             eventBus: this.pdfEventBus,
             externalLinkRel: 'noopener noreferrer nofollow', // Prevent referrer hijacking
-            externalLinkTarget: this.pdfjsLib.LinkTarget.BLANK, // Open links in new tab
+            externalLinkTarget: Browser.isIE() ? this.pdfjsLib.LinkTarget.BLANK : this.pdfjsViewer.LinkTarget.BLANK, // Open links in new tab
```

Comment:
@karelee7 Have we tested opening links from PDFs in modern/ie browsers?

Reply:
Ah links...I don't think so, but let me check now

Reply:
Links look fine! I can click on links/go to external urls from the PDF in modern browsers :)